File: gpo10/Alumno.py
class Alumno:
	def __init__(self, nombre = "nobody", edad=0, numCuenta="000000000"):
		print("Construyendo Alumno")
		self.nombre =nombre
		self.edad = edad
		self.numCuenta = numCuenta



fabiola = Alumno("fabiola")

print (fabiola.nombre)

File: gpo10/Alumno.java
public class Alumno{
	String nombre;
	int edad;
	String numCuenta;

	public Alumno(){
		nombre = "nobody";
		edad = 0;
		numCuenta ="000000000";
		System.out.println("Construyendo Alumno");
	}

	public Alumno(String nombre, int edad, String numCuenta){
		this.nombre = nombre;
		this.edad = edad;
		this.numCuenta = numCuenta;
	}

	public String toString(){
		return "Yo soy " + this.nombre + " y mi numero de cuenta es: "+numCuenta;
	}

	//setter y  getter
	void setNombre(String nombre){
		this.nombre = nombre;
	}

	String getNombre(){
		return this.nombre;
	}

	//metodo generico
	
	void estudia(){
		System.out.println("Alumno estudiando... ay ajaaa!!");
	}

}













